src/main/java/org/apache/commons/text/similarity/RegexTokenizer.java

```diff
@@ -26,12 +26,14 @@
 
 /**
  * A simple word tokenizer that utilizes regex to find words. It applies a regex
- * {@code}(\w)+{@code} over the input text to extract words from a given character
+ * {@code (\w)+} over the input text to extract words from a given character
  * sequence.
  *
  * @since 1.0
  */
 class RegexTokenizer implements Tokenizer<CharSequence> {
+    /** The whitespace pattern. */
+    private static final Pattern pattern = Pattern.compile("(\\w)+");
```

Comment:
s/pattern/PATTERN to fix checkstyle?

Reply:
Sorry. Ran `mvn test` but not the checkstyle.